scripts/synapse_port_db

```diff
@@ -139,39 +140,6 @@ class Store(
     UserDirectoryBackgroundUpdateStore,
     StatsStore,
 ):
-    def __init__(self, db_conn, hs):
-        super().__init__(db_conn, hs)
-        self.db_pool = hs.get_db_pool()
-
-    @defer.inlineCallbacks
-    def runInteraction(self, desc, func, *args, **kwargs):
```

Comment:
there must have been a reason this was here: any idea what it was, and why it no longer applies?

Reply:
We didn't inherit from SQLBaseStore previously, instead we just cherry-picked stuff out by the looks of things

Reply:
`ClientIpBackgroundUpdateStore` inherits from `SQLBaseStore` ?

Reply:
Yup, but we didn't inherit from that when the script was first written and `runInteraction` was first added. We later started inheriting from those stores without removing the `runInteraction` copy

Reply:
hmm right.